packages/apollos-church-api/src/data/content-item-slugs/data-source.js

```diff
@@ -0,0 +1,22 @@
+import RockApolloDataSource from '@apollosproject/rock-apollo-data-source';
+import ApollosConfig from '@apollosproject/config';
+
+const { ROCK } = ApollosConfig;
+
+export default class ContentChannelItemSlug extends RockApolloDataSource {
+  resource = 'ContentChannelItemSlugs';
+
+  getShareURL = async (id, contentChannelId, dataSources) => {
+    try {
+      const contentChannel = await dataSources.ContentChannel.getFromId(
```

Comment:
Instead of needing to pass the datasource through, you can do 
```
this.context.dataSources.ContentChannel 
```
:D

Reply:
Good! Because I hated that I had to do that. 🎉